sw/device/silicon_creator/lib/chip_info.h

```diff
@@ -9,19 +9,51 @@
 
 #include "sw/device/lib/base/macros.h"
 
+/**
+ * A truncated commit hash from the open-source OpenTitan repo that can be
+ * used to reproduce the ROM binary.
+ */
+typedef struct chip_info_scm_revision {
+  /**
+   * Least significant word of the truncated commit hash.
+   */
+  uint32_t scm_revision_low;
+  /**
+   * Most significant word of the truncated commit hash.
+   */
+  uint32_t scm_revision_high;
+} chip_info_scm_revision_t;
+
 typedef struct chip_info {
-  // A truncated commit hash from the open-source OpenTitan repo that can be
-  // used to reproduce the ROM binary.
-  uint64_t scm_revision;
+  /**
+   * Truncated commit hash.
+   */
+  chip_info_scm_revision_t scm_revision;
+  /**
+   * Chip info format version.
+   *
+   * The chip info struct is placed at the end of the ROM. Placing this field at
+   * the end of the struct gives later boot stages a fixed address to look for
+   * this field.
```

Comment:
nit: Add a redundant comment mentioning that the version word lands at the last addressable ROM address, and reference linker script for additional details.